tests/AuthenticatedUser.py

```diff
@@ -101,22 +101,28 @@ def testEditWithAllArguments(self):
     def testEmails(self):
         self.assertEqual(
             self.user.get_emails(),
-            ["vincent@vincent-jacques.net", "github.com@vincent-jacques.net"],
+            [
+                EmailData(email="vincent@vincent-jacques.net", primary=True, verified=True, visibility="private"), 
```

Comment:
Surely this is NameError? You need to call the object returned. Have a look at other examples that call a method using assertListKeyEquals, and assert on .email only -- maybe checking the first item has private visibility and so on.

Comparing complex data structures is not easy with base unittest.

Reply:
I am checking only for emails. Maybe checking for one "primary" would make sense, but nothing more.